Packs/MicrosoftExchangeOnline/Integrations/EWSO365/README.md

```diff
@@ -206,29 +237,33 @@ Impersonation rights required. In order to perform actions on the target mailbox
 
 ```
 
-### 2\. Delete the attachments of an item
+</details>
+<details>
+<summary><h3 style={{display: 'inline'}}>ews-delete-attachment</h3></summary>
+
+### ews-delete-attachment
 
-* * *
+#### Description
 
 Deletes the attachments of an item (email message).
 
-##### Required Permissions
+#### Permissions
 
-Impersonation rights required. In order to perform actions on the target mailbox of other users, the service account must be part of the ApplicationImpersonation role.
+Impersonation rights required. In order to perform actions on the target mailbox of other users, the service account must be part of the _ApplicationImpersonation_ role.
 
-##### Base Command
+#### Limitations
 
-`ews-delete-attachment`
+No known limitations.
 
-##### Input
+#### Inputs
 
 |**Argument Name**|**Description**|**Required**|
 |--- |--- |--- |
 |item-id|The ID of the email message for which to delete attachments.|Required|
 |target-mailbox|The mailbox in which this attachment was found. If empty, the default mailbox is used. Otherwise, the user might require impersonation rights to this mailbox.|Optional|
 |attachment-ids|A CSV list (or array) of attachment IDs to delete. If empty, all attachments will be deleted from the message.|Optional|
```

Comment:
For consistency, we should use "A comma-separated list (or array).....
Update in all places.